bower.json

```diff
@@ -18,8 +18,10 @@
     "purescript-console": "^2.0.0",
```

Comment:
I was wandering about it too.
But maybe sometimes it is useful to log some stuff on production. So maybe it is ok?